QEfficient/generation/text_generation_inference.py

```diff
@@ -269,6 +269,7 @@ def cloud_ai_100_exec_kv(
     prompts_txt_file_path: Optional[str] = None,
     device_id: Optional[List[int]] = None,
     generation_len: Optional[int] = None,
+    CCL: Optional[List[int]] = None,
```

Comment:
nit: can we keep it in small letter, maybe `enable_ccl `or something? again to keep consistent with other args